List dependencies for deb/rpm install (#1596)
hazelcast/hz-docs: docs/modules/getting-started/pages/install-hazelcast.adoc

= Installing Hazelcast {open-source-product-name}

> :description: You can install the {open-source-product-name} of Hazelcast, using the command line interface (CLI), Docker, the binary, or use as Java library.

Reviewer: ```suggestion
:description: You can install the {open-source-product-name} of Hazelcast using a package manager, Docker, or the binary. Alternatively, you can add Hazelcast as a dependency in your Java project.
```

I don't think "the CLI" is quite right (which CLI?) and the docs otherwise refer to a package manager, and it's hard to get the Java library use case into the same sentence and have it read well.

+
--

> Use either of these package managers to install Hazelcast, depending on your Linux distribution:

Reviewer: ```suggestion
You can use either of these package managers to install Hazelcast, depending on your Linux distribution:
```

+
--

Use either of these package managers to install Hazelcast, depending on your Linux distribution:

.Debian

[source,shell,subs="attributes+"]
----
# install necessary tools
sudo apt-get install -y --no-upgrade wget gpg coreutils \
  && wget -qO - https://repository.hazelcast.com/api/gpg/key/public | gpg --dearmor | sudo tee /usr/share/keyrings/hazelcast-archive-keyring.gpg > /dev/null \
  && echo "deb [signed-by=/usr/share/keyrings/hazelcast-archive-keyring.gpg] https://repository.hazelcast.com/debian stable main" | sudo tee -a /etc/apt/sources.list \
  && sudo apt update && sudo apt install hazelcast={os-version}
----

.RPM
[source,shell,subs="attributes+"]
----

> # install necessary tool

Reviewer 2: I don't think these comments add value in isolation.

+
--

Use either of these package managers to install Hazelcast, depending on your Linux distribution:

.Debian

[source,shell,subs="attributes+"]
----
# install necessary tools
sudo apt-get install -y --no-upgrade wget gpg coreutils

> wget -qO - https://repository.hazelcast.com/api/gpg/key/public | gpg --dearmor | sudo tee /usr/share/keyrings/hazelcast-archive-keyring.gpg > /dev/null

Author: good point, eb89c4e60810dc55a10569ca65b6b89b4ea208b9

echo "deb [signed-by=/usr/share/keyrings/hazelcast-archive-keyring.gpg] https://repository.hazelcast.com/debian stable main" | sudo tee -a /etc/apt/sources.list
sudo apt update && sudo apt install hazelcast={os-version}
----

.RPM
[source,shell,subs="attributes+"]
----

> # install necessary tool

Reviewer 2: ```suggestion
# install necessary tools
```

As on the other one.

Author: RPM requires only wget (one thing)

Reviewer 2: 🤦
